templates/python/custom-copilot-basic/src/bot.py.tpl

```diff
@@ -48,14 +49,10 @@ bot_app = Application[TurnState](
         bot_app_id=config.APP_ID,
         storage=storage,
         adapter=TeamsAdapter(config),
-        ai=AIOptions(planner=planner),
+        ai=AIOptions(planner=planner, enable_feedback_loop=True),
     )
 )
 
-@bot_app.conversation_update("membersAdded")
-async def on_members_added(context: TurnContext, state: TurnState):
-    await context.send_activity("How can I help you today?")
```

Comment:
Can you share why we want to remove this welcome message?

Reply:
because welcome message will refresh the zero-prompt page